src/server/game/DataStores/DBCStores.cpp

```diff
@@ -157,6 +159,16 @@ DBCStorage <SkillLineEntry> sSkillLineStore(SkillLinefmt);
 DBCStorage <SkillLineAbilityEntry> sSkillLineAbilityStore(SkillLineAbilityfmt);
 DBCStorage <SkillRaceClassInfoEntry> sSkillRaceClassInfoStore(SkillRaceClassInfofmt);
 SkillRaceClassInfoMap SkillRaceClassInfoBySkill;
+
+static AbilitiesBySkillLine sAbilitiesBySkillLine;
+std::vector<SkillLineAbilityEntry const*> const* GetSkillLineAbilitiesBySkill(uint32 skillLine)
+{
+    auto i = sAbilitiesBySkillLine.find(skillLine);
```

Comment:
you can use Trinity::Containers::MapGetValuePtr

return Trinity::Containers::MapGetValuePtr(sAbilitiesBySkillLine, skillLine);